yarn.lock

```diff
@@ -5636,9 +5636,6 @@ eslint-plugin-jsx-a11y@^6.0.3:
     has "^1.0.3"
     jsx-ast-utils "^2.0.1"
 
-"eslint-plugin-material-ui@file:packages/eslint-plugin-material-ui":
```

Comment:
Any idea what happened here? This is probably responsible for the failed checkout CI job.